src/diffusers/models/unets/unet_3d_blocks.py

```diff
@@ -494,8 +494,8 @@ def __init__(
             )
             attentions.append(
                 Transformer2DModel(
-                    out_channels // num_attention_heads,
                     num_attention_heads,
+                    out_channels // num_attention_heads,
```

Comment:
Sorry, why isn't it backwards compatible? None of the args in the class init are being changed right?  

Is it because users might be passing in `num_attention_heads` with the wrong values?  The 3D blocks and the 3D UNet are only used with the Text to Video Synth and I2VGenXL model in the library.

Reply:
Same with `get_up_block` and `get_down_block`. We're only changing the number being passed in to the `num_attention_heads` argument.

Reply:
> The 3D blocks and the 3D UNet are only used with the Text to Video Synth and I2VGenXL model in the library.

yes but it is our public API and we have to assume it's been used publically

Reply:
Even getting specific with the imports doesn't return a lot of results
https://github.com/search?q=%22from+diffusers.models.unet_3d_blocks+import+CrossAttnDownBlock3D%22+language:Python+&type=code

Reply:
```sh
python -c "from diffusers import CrossAttnDownBlock3D"
```

doesn't work, so strictly speaking CrossAttnDownBlock3D is **not** considered part of the public API. Also I don't think it's used that much so IMO it's ok to change it here (while making sure though we note it correctly in the PR description).